src/app.ts

```diff
@@ -0,0 +1,35 @@
+import {IncomingMessage, ServerResponse} from "http";
+import {ListenOptions} from "net";
+
+const http = require('http')
+const fs = require('fs')
+const filePath: string = './text/exampleFile.txt'
+const contentOptions: ListenOptions = {
+    host: 'localhost',
+    path: '/CONTENT',
+    port: 3000
+}
+let fileContent: string = '';
```

Comment:
do not save all your file inside single string. send parts of this file each time. read about res.send, res.end etc